machine/v1beta1/types_vsphereprovider.go

```diff
@@ -172,6 +179,15 @@ type NetworkDeviceSpec struct {
 	AddressesFromPools []AddressesFromPool `json:"addressesFromPools,omitempty"`
 }
 
+// VSphereDisk describes additional disks for vSphere.
+type VSphereDisk struct {
+	// The device name exposed. Used to identify the disk.
+	// +optional
+	DeviceName *string `json:"deviceName,omitempty"`
+	// SizeGB is the size of the disk (in GB).
+	SizeGB int64 `json:"sizeGb"`
```

Comment:
Can you add an explicit +kubebuilder:validation:Required marker please

Reply:
ok.  I will add.  thanks.